### PR TITLE
feat: maps tooltip

### LINKED ISSUE #1715: Maps: Add ability to send tooltips in with Config
In giraffe, we have added a new property on PointMapLayer to be able to send in tooltipColumns. This issue is to update the UI to pull in the new properties.

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #1715

The ui side of tooltip columns

### CODE REVIEW

File: src/views/helpers/index.ts [hunk 1]
```diff
@@ -403,6 +403,14 @@ const NEW_VIEW_CREATORS = {
           colorField: '_value',
           colors: DEFAULT_THRESHOLDS_GEO_COLORS,
           isClustered: false,
+          tooltipColumns: [
```
Comment (Palakp41) [inline src/views/helpers/index.ts:406]: I don't think every dataset provided will have these columns. A lot of the times we filter out the columns and the column names like `s2_cell_id` can be named differently. What happens if this array is empty as it should be unless populated right?

**Review by Palakp41 — CHANGES_REQUESTED** [review]
I think you need to specify the tooltip columns to be an empty array, or with just `start`, `stop`, and `time` columns, which we know always exist.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

File: src/visualization/types/Map/view.tsx [hunk 2]
```diff
@@ -122,6 +124,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         colorField: '_value',
         colors: DEFAULT_THRESHOLDS_GEO_COLORS,
         isClustered: false,
+        tooltipColumns: tooltipColumns,
```
Comment (Palakp41) [inline src/visualization/types/Map/view.tsx:127]: So the code never reaches here, hence the tooltip columns is always `[]`. This is a safe fail guard against layers. You might want to do same thing as we are doing for colors in the line 133.

**Pull request closed without merging**